spec/features/managing_editor_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+RSpec.feature "Managing editor permissions" do
+  background do
+    given_i_dont_have_the_managing_editor_permission
+  end
+
+  scenario "published edition" do
+    when_there_is_a_published_edition
+    and_i_visit_the_summary_page
+    and_i_click_on_withdraw
+    then_i_see_a_message_to_ask_my_managing_editor_to_withdraw_content
+  end
+
+  scenario "withdrawn edition" do
+    when_there_is_a_withdrawn_edition
+    and_i_visit_the_withdrawn_document_summary_page
+    and_i_click_on_change_public_explanation
+    then_i_see_a_message_to_ask_my_managing_editor_to_withdraw_content
+  end
```

Comment:
Hmm since there's only one thing in common between these (the `given_i_dont_have_the_managing_editor_permission`) it might not actually be that nice that these share the same file. 

I think it's also a little counter intuitive that this is called managing editior permissions when it's all about the lack of them.

It doesn't seem immediately obvious what the best way to solve this is that is consistent with our codebase. 

I'd probably suggest that the most consistent way is "withdraw a document without managing editor permission" and "edit a withdrawal explanation without managing editor permission". Does feel like we end up with an awful lot of feature tests doing that but that probably plays into the wall of pain card of: https://trello.com/c/sQ7Go4dm/570-hard-to-work-out-how-to-test-off-the-happy-path

Reply:
All the other tests are updated to include the managing editor permissions, so maybe that's enough to test that the permission works, and we scrap this file?